### DISCUSSION TITLE
Custom Notification Sound

### DISCUSSION THREAD

**Discussion by drpayyne** [community/community#5613] [Mobile]
_Tiny suggestion but I think it would be very very useful._

Can we have custom notification sounds? It's currently the default iOS notification sound, which can be changed for the whole system, but not just for GitHub mobile. Can we get this feature? It would be good to have a different notification sound for GitHub mobile so that its notifications are distinguishable without checking the phone.

I'm on iOS, so I'm not sure if this applies to Android too.

**Comment by aconsuegra** [comment]
Hi @drpayyne. Thanks for your suggestion! I have created an internal issue to track this feature request, and we will evaluate it soon.
